src/sentry/static/sentry/app/views/settings/components/forms/formField/formFieldWrapper.jsx

```diff
@@ -6,32 +6,36 @@ import SettingsInputField from '../styled/input';
 import SettingsTextAreaField from '../styled/textarea';
 
 const FormFieldWrapper = styled(({highlighted, inline, ...props}) => <Flex {...props} />)`
-  padding: 15px 0 15px 20px;
+  padding: 0.9em 0 0.9em 1.3em;
```

Comment:
this should have no impact on the design elsewhere, but enables me to control the size of the line items from the fine tuning scope.